manifests/k/Kingsoft/KDocs/3.2.0/Kingsoft.KDocs.installer.yaml

```
@@ -0,0 +1,67 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Kingsoft.KDocs
+PackageVersion: 3.2.0
```

This installs 3.0.4 for me
```suggestion
PackageVersion: 3.0.4
```